### ISSUE TITLE
Datasets

### ISSUE THREAD

**Issue by victorca25** [SystemErrorWang/White-box-Cartoonization#20]
Very interesting work and results. I have a couple of questions. From the paper:

> For cartoon images, we collect 10000 images from animations for the human face and 10000 images for landscape. Producers of collected animations include Kyoto animation, P.A.Works, Shinkai Makoto, Hosoda Mamoru, and Miyazaki Hayao.

Can you share more details about how were these images collected? 10000 images of what size? Any particular algorithm of what images are used and what images are discarded from the animations? Any kind of balance in the dataset? Buildings, nature, etc.

Another question, in different parts of the paper, the code and the readme VGG19 and VGG16 appears to be used interchangeably, but they are not the same. Which one was used VGG19 or VGG16? Was it fine-tuned in any way or only used the stock pretrained model to extract the high level features? 

Lastly, where in the code is the style interpolation used? Or is it only for inference to interpolate models trained with different loss weights?

**Comment by SystemErrorWang** [comment]
@victorca25 animation face are collected from kyoto animate and pa works, we use animate facial detector to crop them and manually cleaned the dataset. VGG network is pretrained on the imagenet, we use vgg19 which is described in the training data and you can download it from the google drive link in readme. the style interpolation is only used during inference.

**Comment by victorca25** [comment]
Thanks for the quick answer.

What about the landscape images? Is it 10000 full images that are later cropped or is it 10000 cropped images? 

Regarding the VGG, the paper only mentions VGG16, that's where the question came from, trying to identify the differences. BTW, the link in the readme is broken, it leads to a 404. But if it's a stock VGG it should not be a problem.

**Comment by SystemErrorWang** [comment]
@victorca25 VGG16 is a typo in the paper, I am sorry for it. for landscape images, real world photos are from cyclegan dataset (6227), cartoon images are collected from Shinkai Makoto, Miyazaki Hayao and Hosoda Mamoru films

**Comment by victorca25** [comment]
Perfect, thank you very much for your answers!
